CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ### 0.7.1 (Next)
 
 * Your contribution here.
+* [#82](https://github.com/dblock/slack-ruby-client/pull/82): Fix usergroups.users.{list,update} and files.comments.{add,edit,delete} APIs
```

Comment:
Make this look like the other entries please, quote the methods and add your `- name.` at the end.